php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -49,7 +49,7 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
         $this->commentID = $commentID;
         $this->page      = $page;
 
-        $this->form = new \HTML_Quickform('test_form');
+        $this->form = new \LorisForm('test_form');
```

Comment:
I believe that [line 426](https://github.com/jstirling91/Loris/blob/19d893e5ea175f82f18b75b8e022b8b6de6a9cf8/php/libraries/NDB_BVL_Instrument_LINST.class.inc#L426) in this file also needs to be updated